build/foss/Cargo.toml

```diff
@@ -9,4 +9,4 @@ crate-type = ["staticlib"]
 
 [dependencies]
 norddrop = { git = "https://github.com/NordSecurity/libdrop", tag = "v5.4.0" }
-telio = { git = "https://github.com/NordSecurity/libtelio", tag= "v4.1.2" }
+telio = { git = "https://github.com/NordSecurity/libtelio", branch= "main" }
```

Comment:
This is temporary since no libtelio version with this functionality has been released.
I'll change this as soon as the appropriate release is out, before merging ofc

Reply:
No longer relevant. I'm using the tagged version